Sources/SWBTaskConstruction/ProductPlanning/ProductPlanner.swift

```diff
@@ -467,15 +467,11 @@ extension StandardTarget
         for taskProducerExtension in taskProducerExtensions {
             for factory in taskProducerExtension.unorderedPostSetupTaskProducers {
                 createNewPhaseNode(factory.name)
-                taskProducers.append(factory.createTaskProducer(taskProducerContext, startPhaseNodes: [setupEndPhaseNode], endPhaseNode: endPhaseNode))
+                taskProducers.append(factory.createTaskProducer(taskProducerContext, startPhaseNodes: [setupEndPhaseNode, buildPhasesEndNode], endPhaseNode: endPhaseNode))
```

Comment:
Some post-setup task producers (for example, StubBinaryTaskProducerFactory) will be injected via extension points here but shouldn't be ordered relative to buildPhasesEndNode. I think we might need to introduce an unorderedPostBuildPhasesTaskProducers counterpart to taskProducerExtension.unorderedPostSetupTaskProducers so we can differentiate between the two use cases